cli/up.go

```diff
@@ -0,0 +1,26 @@
+package cli
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var (
+	upCmd = &cobra.Command{
+		Use:   "up <group> [scale]",
+		Short: "Bring a group up",
+		Long:  "Submit a group, with an optional scale, and start it",
+		Run:   upRun,
+	}
+)
+
+func upRun(cmd *cobra.Command, args []string) {
+	submitRun(cmd, args)
```

Comment:
well, I think the expectation of `inagoctl up mygroup 4` would be that afterwards there are 4 slices. We probably can involve @marians here. 

What does this mean it there are already 2 slices? Do we just want to submit 2 slices and then start all 4?